packages/hardhat-core/src/internal/core/errors-list.ts

```diff
@@ -963,9 +963,9 @@ If you aren't overriding compilation-related tasks, please report this as a bug.
     INVALID_SOURCE_NAME_BACKSLASHES: {
       number: 1002,
       message:
-        "Invalid source %name%. The source name uses backslashes (\\) instead of slashes (/).",
+        "Invalid source %name%. The source name uses backslashes (\\\\) instead of slashes (/).",
```

Comment:
I tried this in a node console

```
console.log("Invalid source %name%. The source name uses backslashes (\\\\) instead of slashes (/).")
```

and it printed: Invalid source %name%. The source name uses backslashes (\\) instead of slashes (/).

Reply:
At first I was confused by your comment, but then I reproduced what you're saying, and I saw the problem, and then I edited your comment in order to surround your output with backticks (`), and now the problem is visible in your comment as well.

I wrote this code before you explained to me the difference between `message` and `description`, but now that you've explained it to me, I understand the reason for your comment here.

Addressed in https://github.com/nomiclabs/hardhat/pull/1554/commits/5be5b9d9e422488f3b7dc246b29888265b6e8e1f